sportsref/ncaaf/players.py

```diff
@@ -147,8 +152,17 @@ def rushing_and_receiving(self):
         :returns: Pandas DataFrame with stats.
         """
         doc = self.getDoc()
-        table = doc('#rushing')
+        table = doc('#all_rushing')
```

Comment:
whose page are you using to look at the HTML? The pages I've looked at are all either 'table#rushing' or 'table#receiving', I haven't seen any all_rushing or all_receiving